src/tests/mod.rs

```diff
@@ -106,11 +164,16 @@ fn get_config() -> CliArgs {
         config_hash_version: String::from(SN_OS_CONFIG_HASH_VERSION),
         app_chain_id: String::from(APP_CHAIN_ID),
         fee_token_address: String::from(FEE_TOKEN_ADDRESS),
-        cross_chain_wait_time: 120,
+        native_fee_token_address: String::from(NATIVE_FEE_TOKEN_ADDRESS),
```

Comment:
So this is the default config for `CliArgs`? We should probably just `implement Default`

Reply:
created